### DISCUSSION TITLE
Lista de PSPs suportando Pix Automático

### DISCUSSION THREAD

**Discussion by rubenskuhl** [bacen/pix-api#608] [General]
Atualização: 25.09.2024

No #76 eu mantenho uma lista de PSPs recebedores implementando API Pix, e no #480 um de que tem envio de Pix via API. 

Este tópico vai agregar PSPs recebedores ofertando Pix Automático, o recurso que os PSPs pagadores terão que implementar a partir de 06.2025. Como haverá tanto oferta via API quanto via arquivo padronizado, fez mais sentido ter uma lista diferente da de API Pix. 

Neste momento a lista se baseará em intenções de suporte a esse recurso; conforme ambientes de homologação e produção forem surgindo, a lista irá se moldando à realidade de quem oferece, e o quê. 
A lista é ordenada por data de inclusão, não significando qualificação ou recomendação conforme a ordem. 

1. PSP: Efí S.A. - IP
ISPB: 09089356
Pix Automático via API Pix: Sim (4 jornadas)
Pix Automático via arquivo: ?

**Comment by GuerreiroPUSHINPAY** [comment]
Para todos ! Segue opção de PIX via API - pushinpay.com.br 
Melhores taxas do mercado

**Reply by rubenskuhl** [reply]
> pushinpay.com.br

Não é um membro do arranjo Pix, segundo a lista do Banco Central.

**Reply by GuerreiroPUSHINPAY** [reply]
Nosso fornecedor de serviços PIX é, somos um prestador de serviços em meio de pagamento e utilizamos no momento 3 BANCOS que estão listados no Banco Central.
Ficamos a disposição para tirar qualquer dúvida.

**Reply by rubenskuhl** [reply]
> Nosso fornecedor de serviços PIX é, somos um prestador de serviços em meio de pagamento e utilizamos no momento 3 BANCOS que estão listados no Banco Central. Ficamos a disposição para tirar qualquer dúvida.

Isso coloca vocês como um gestor de pagamentos, na regulamentação sendo criada pelo Banco Central. Mas que se sair no modelo que o BACEN está sugerindo, irão existir zero... pq ela coloca tanta culpa nos membros do arranjo que ninguém vai querer o risco de ser o PSP desse tipo de prestador. 

De qualquer forma, os prestadores listados em #76 e neste tópico são apenas membros do arranjo Pix, não incluindo gestores de pagamento.

**Reply by GuerreiroPUSHINPAY** [reply]
Perfeito Rubens, somos gestores de pagamento com contratos de compliance bem regidos para podermos executar esse tipo de operação. Nos diferenciamos pela humanização no atendimento e velocidade nas integrações. Mas entendo que o tópico é apenas para os membros. Aproveitei o título para apresentar a empresa que pode prestar o serviço de qualidade com taxas ótimas.

**Comment by yvesmariano** [comment]
Banco Inter - 4 Jornadas
https://developers.inter.co/references/pix-automatico

**Comment by yvesmariano** [comment]
BTG Pactual
https://developers.empresas.btgpactual.com/docs/pix-automático

**Comment by yvesmariano** [comment]
Transfeera
https://docs.transfeera.dev/docs/pix-automatico-como-funciona

**Reply by rubenskuhl** [reply]
Essa está suspensa pelo Banco Central...

**Comment by GuerreiroPUSHINPAY** [comment]
Segue nossa documentação
https://app.theneo.io/pushinpay/pix/criar-pix
Melhores taxas de PIX do mercado e trabalhamos com MULTI-adquirentes

Soluções também para CHECKOUT e BOLETO!
